tigers-lottery/app/src/main/java/com/example/tigers_lottery/CreateEntrantProfileActivity.java

```diff
@@ -107,6 +103,11 @@ public void onClick(View view) {
                     date = calendar.getTime();
                 }
 
+                if(dbHelper.validatingUserProfileInput(firstnameEditText.getText().toString(), lastnameEditText.getText().toString(), emailEditText.getText().toString(), date)) {
+                    Toast.makeText(CreateEntrantProfileActivity.this, "Every field except for Phone Number must be filled!", Toast.LENGTH_LONG).show();
+                    return;
+                }
```

Comment:
I dont think this functionality should be implemented in dbhelper, you can create a local method to check for that